### PR TITLE
Editionable landing pages

### PR DESCRIPTION
**Pull request by KludgeKML**
https://trello.com/c/xZrRMK6P

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

Follow [these steps](https://guides.rubyonrails.org/upgrading_ruby_on_rails.html) if you are doing a Rails upgrade.

### CODE REVIEW

File: app/presenters/publishing_api/landing_page_presenter.rb [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        content.merge!(PayloadBuilder::FirstPublishedAt.for(item))
+      end
+    end
+
+    def links
+      # TODO: Previously, this presenter was sending all links to the
+      # Publishing API at both the document level, and edition
+      # level. This is probably redundant, and hopefully can be
+      # improved.
+      edition_links
```
Comment (richardTowers) [inline app/presenters/publishing_api/landing_page_presenter.rb:38]: Maybe let's drop this comment (which we copied over from another thingy) and just do:

```
{}
```

like in edition links, but not coupling them together.

File: test/unit/app/presenters/publishing_api/landing_page_presenter_test.rb [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+  test "it presents the schema_name as landing_page" do
+    assert_equal "landing_page", @presented_content[:schema_name]
+  end
+
+  test "it presents the document type as landing_page" do
+    assert_equal "landing_page", @presented_content[:document_type]
+  end
+
+  test "it presents the global process wide locale as the locale of the fatality_notice" do
```
Comment (richardTowers) [inline test/unit/app/presenters/publishing_api/landing_page_presenter_test.rb:62]: s/fatality_notice/landing_page/

File: app/presenters/publishing_api/landing_page_presenter.rb (hunk 1 above)
Comment (KludgeKML) [reply app/presenters/publishing_api/landing_page_presenter.rb]: Fixed!

File: test/unit/app/presenters/publishing_api/landing_page_presenter_test.rb (hunk 2 above)
Comment (KludgeKML) [reply test/unit/app/presenters/publishing_api/landing_page_presenter_test.rb]: Fixed!

**Review by richardTowers — APPROVED** [review]

**Pull request merged**